package.json

```diff
@@ -1,7 +1,13 @@
 {
-  "name": "my-app",
-  "version": "0.1.0",
+  "name": "simple-rest-api",
+  "version": "0.0.1",
+  "license": "MIT",
   "private": true,
```

Comment:
In all our template this key is true, we have a project that we don't have it, 
I was asking about in the past and the decision was to keep it like that.

Reply:
private in this sense means don't allow `npm publish`, and since this is not really a package, it makes perfect sense. See https://docs.npmjs.com/files/package.json#private